src/pages/finalize.js

```diff
@@ -86,17 +88,23 @@ export const finalizeTemplate = (data, specimenData) => {
                     </div>
                 </div>
                 </br>
-                <div class="form-group row">
-                    <div class="col-auto">
-                        <button class="btn btn-outline-danger" type="button" data-connect-id="${data.Connect_ID}" id="returnToCollectProcess" data-master-specimen-id="${specimenData['820476880']}">${getWorkflow() === 'research' ? 'Return to Collection Data Entry' : 'Return to Collection Data Entry'}</button>
-                    </div>
-                    <div class="ml-auto">
-                        <button class="btn btn-outline-warning" data-connect-id="${data.Connect_ID}" data-master-specimen-id="${specimenData['820476880']}" type="button" id="finalizedSaveExit">Exit</button>
-                    </div>
-                    <div class="col-auto">
-                        <button class="btn btn-outline-primary modal-open" data-modal-id="modal1" data-connect-id="${data.Connect_ID}" data-master-specimen-id="${specimenData['820476880']}" type="submit" id="finalizedContinue">Review Complete</button>
+                ${flag === true ? 
+                    `<div class="form-group row">
+                        <div class="col-auto">
+                            <button class="btn btn-outline-danger" type="button" id="returnToSpecimenSearch">Return to Search</button>
+                        </div>
                     </div>
+                    ` :`<div class="form-group row">
+                <div class="col-auto">
+                    <button class="btn btn-outline-danger" type="button" data-connect-id="${data.Connect_ID}" id="returnToCollectProcess" data-master-specimen-id="${specimenData['820476880']}">${getWorkflow() === 'research' ? 'Return to Collection Data Entry' : 'Return to Collection Data Entry'}</button>
+                </div>
+                <div class="ml-auto">
+                    <button class="btn btn-outline-warning" data-connect-id="${data.Connect_ID}" data-master-specimen-id="${specimenData['820476880']}" type="button" id="finalizedSaveExit">Exit</button>
```

Comment:
Maybe we can `specimenData['820476880']` replace the concept Id and reference the [fieldToConceptIdMapping.js](https://github.com/episphere/biospecimen/blob/dev/src/fieldToConceptIdMapping.js#L72-L73) file